packages/prompts/src/common.ts

```diff
@@ -2,6 +2,7 @@ import type { Readable, Writable } from 'node:stream';
 import type { State } from '@clack/core';
 import isUnicodeSupported from 'is-unicode-supported';
 import color from 'picocolors';
+import type { ProgressOptions } from './progress-bar.js';
```

Comment:
I don't really like the fact that `common.ts` is dependent of `progress-bar.(t|j)s`.

But I don't know if best solution exist :thinking: 

----

Maybe the `S_PROGRESS_CHAR` can be left in the `progress-bar` file, but the `s` function need to be exported then.

Reply:
we should just keep them in the progress bar module, since nothing else needs them

if they were shared/"common", they'd be individual constants like the ones above

we can export `s` as long as we give it a real name, like `unicodeFallback` or something